src/actions/__tests__/authActions.spec.js

```diff
@@ -125,4 +140,104 @@ describe('Auth action tests', () => {
         });
     });
   });
+
+  describe('Async password request action test', () => {
+    let store;
+    const response = {
+      message: 'Hi John, a password reset link has been sent to your email',
+    };
+
+    beforeEach(() => {
+      store = mockStore({});
+    });
+
+    afterEach(() => {
+      nock.cleanAll();
+    });
+
+    it('send password link successfully', () => {
+      nock(url)
+        .post('/users/passwordReset')
+        .reply(200, response);
+
+      return store.dispatch(requestPasswordLink({}))
+        .then(() => {
+          expect(store.getActions()).toMatchSnapshot();
+        });
+    });
+
+    // it('should throw error as expected', () => {
+    //   const errorResponse = {
+    //     err: {
+    //       response: {
+    //         data: {},
+    //       },
+    //     },
+    //   };
+    //   nock(url)
+    //     .post('/users/passwordReset')
+    //     .reply(400, errorResponse.err);
+
+    //   return store.dispatch(requestPasswordLink({}))
+    //     .then(() => {
+    //       expect(store.getActions()).toMatchSnapshot();
+    //     });
+    // });
+  });
```

Comment:
why do you have commented out codes? @Hadeneekeh

Reply:
I will fix it now

Reply:
Fixed